packages/commonwealth/server/scripts/chainEventsConsumer.ts

```diff
@@ -76,6 +80,10 @@ const setupChainEventListeners = async (wss: WebSocket.Server):
 
   // feed the events into their respective handlers
   async function processClassicEvents(event: CWEvent): Promise<void> {
+    if (!RmqCWEvent.isValidMsgFormat(event)) {
+      throw RmqCWEvent.getInvalidFormatError(event);
```

Comment:
Do we really want to stop the program here with `throw` or just log? What do you think is best? Might be good to write a test case for this. We wouldn't want an invalid format to take this process down in prod.

Reply:
This doesn't take down the process. This error is caught by the RabbitMQController and the recovery strategy depends on what type of error is throw. This is why the RmqCWEvent class/namespace is important because it defines the error that has a specific effect i.e. this error type immediately tells RabbitMQ to route the offending message to a dead letter queue rather than attempt to process it or requeue it multiple times.